build/yaml/crd/nsx.vmware.com_ippools.yaml

```diff
@@ -0,0 +1,129 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: ippools.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: IPPool
+    listKind: IPPoolList
+    plural: ippools
+    singular: ippool
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
```

Comment:
`v1alpha1`: for CRD upgrade from existing IPPool CRD, do we need to update the version here?

Reply:
v1alpha2?